### ISSUE TITLE
Sortable table column icons below min contrast ratio (WCAG Level AA)

### ISSUE THREAD

**Issue by jrbarnes9** [ONSdigital/design-system#2247]
Increase contrast of sortable table icons to min 3:1

![Screenshot 2022-06-16 at 15 14 53](https://user-images.githubusercontent.com/43346934/174089816-cce7d6ff-cf8b-4fb4-9716-1b64bd58dc66.png)

(Page 68-69 of June 2022 DAC DS report...)

----

### Issue ID: DAC_Colour_Contrast_Non_Text_02

URL: https://ons-design-system-prototypes.netlify.app/prototypes/dac-may2022/table-1.html#tabId2
Page title: Tabs and sortable and scrollable tables
Journey: Journey 5, Step 13

### Screenshot:
Each interactive button for sortable tables contains an up and down pointing triangle. The triangles provide visual indication that there is sort functionality present. Low vision users may have difficulties seeing the up and down triangles due to the colour contrast against adjacent colours being below the minimum.

Foreground: #BCBCBD
Background: #FFFFFF
The contrast ratio is: 1.9:1
1.4.11 Non-text Contrast (AA)
Fail for UI components and graphical objects
### Current code ref(s):
#tabId1 > table > thead > tr > th:nth-child(1) > button 
```
<button aria-label="Sort by ID" type="button" data-index="0" class="ons-table__sort-button">ID <svg id="sort-sprite-id" class="ons-svg-icon " viewBox="0 0 12 19" xmlns="http://www.w3.org/2000/svg" focusable="false" fill="currentColor"> <path class="ons-topTriangle" d="M6 0l6 7.2H0L6 0zm0 18.6l6-7.2H0l6 7.2zm0 3.6l6 7.2H0l6-7.2z"></path> <path class="ons-bottomTriangle" d="M6 18.6l6-7.2H0l6 7.2zm0 3.6l6 7.2H0l6-7.2z"></path> </svg> </button>
```

Low vision comments: “Next to id, title, abbr, legal basis, frequency, date and status are two arrows pointing up and down, they are light grey on a white background which makes them difficult to see due to the contrast, this means users may miss out on these functions.”

### Solution:
Interactive elements that are not disabled must meet a minimum contrast ratio of 3:1
The minimum contrast ratio of 3:1 helps low vision users see non-text elements with minimal difficulty.
